### PR TITLE
Kangz12345/110/replace bus with queued connection

### LINKED ISSUE #110: [FEATURE] Implement Bus using PyQt's queued connection mechanism
### Feature you want to implement
Swift currently uses `Bus` system for broadcasting the messages among the apps.
However, it could be implemented with [queued connection](https://doc.qt.io/qt-6/threads-qobject.html#signals-and-slots-across-threads) mechanism of Qt.

### How the feature is implemented
Current system:
1. An app emits `broadcastRequested` signal with a bus name and a message.
2. Swift routes it to the designated `Bus`, and push the message into its message queue.
3. When the control flow switches to the `Bus`'s queue consumer thread, the message is popped, and the `received` signal of the `Bus` is emitted.
4. Swift routes it to the subscriber apps' `received` signals.

Suggested system:
1. _Same_.
2. `broadcastRequested` signal is connected as `QueuedConnection`, so it is queued in the main thread's queue.
3. When the connected slot is executed by the main thread's event loop, it emits the `received` signal of the subscriber apps.

### Additional context
The reason why we should use the queued system is that an app may abuse the broadcasting.
Imagine:
1. An app broadcasts a message.
2. The app itself subscribes it, and as a result, it broadcasts the same message again.
3. Then it will induce an infinite loop, freezing the GUI.

If we use the queued system, the events between each broadcast can be handled, since the control flow always returns back to the event loop immediately after broadcasting.

See also: #6.

### PR DESCRIPTION
**Pull request by kangz12345**
Please see the related issue #110 for details.

Now there is no _actual buses_.
Instead, only the subscription information is managed.

This closes #110.

### CODE REVIEW

**Comment by kangz12345** [comment]
Note that I didn't change the setup.json file.
It is not a problem, but it is bad since it contains redundant information.
If you think the "bus" part of the setup file should be removed, I will.

**Comment by kangz12345** [comment]
Please note that this PR may _conflict_ with #107, since the methods are removed from `Swift`.
My suggestion is, merge this PR (if there is no problem with this PR) first, and do not test the removed methods in #107.
It may yield low coverage, but is not a problem after merging it.

File: swift/swift.py
Comment (BECATRUE) [inline swift/swift.py:122]: What excellent type!

**Review by BECATRUE — COMMENTED** [review]
The queued connection looks wonderful!

I left a few reviews.

File: swift/swift.py
Comment (kangz12345) [reply swift/swift.py:186]: Oh, I thought that you knew this, because I've got to know this during reviewing your codes before.
As you can see [here](https://www.merriam-webster.com/words-at-play/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage), "cast" is an irregular verb.
Although "-casted" is also used, "-cast" seems the standard English.

Comment (BECATRUE) [reply swift/swift.py:176]: Wow.. I'm poor at English😂 You're right!

**Review by BECATRUE — APPROVED** [review]
It looks good!

**Pull request merged**
